homeassistant/components/vizio/config_flow.py

```diff
@@ -131,6 +132,7 @@ def __init__(self) -> None:
                         user_input[CONF_HOST],
                         user_input.get(CONF_ACCESS_TOKEN),
                         user_input[CONF_DEVICE_CLASS],
+                        session=async_get_clientsession(self.hass, False),
```

Comment:
This seems new, we're no longer validating SSL. Why is that? And why was it introduced ?

Reply:
The change isn't to go from validating SSL to not validating SSL. It's just to pass in HA session in the two places I missed when I first switched to async. You'll see here that we don't validate SSL anywhere since vizio uses an internal cert: https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/vizio/media_player.py#L70

This is technically net new code but functionally is the same so up to you as to whether I include it in this commit or if I should introduce this change in my other PR (#31520)